app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -200,6 +201,8 @@
     @include govuk-media-query($until: tablet) {
       grid-area: 1 / 1;
     }
+
+    margin-top: 2px;
```

Comment:
This line should appear above the media query.

Reply:
This is to apply to desktop as well as mobile, thats why outside of media query

Reply:
Above the media query.

Reply:
second thoughts have removed.... not convinced it aligned correctly